### PR TITLE
Arbo poc

### PR DESCRIPTION
**Pull request by HimanshuRanka**
## Briefly describe the feature or bug that this PR addresses.

Just all the code for the POC to be reviewed and discussed. Might use this itself as a holding branch before release

### CODE REVIEW

**Comment by HimanshuRanka** [comment]
Important to know that we DO NOT WANT to merge this to dev yet, or anytime soon. For all arbo related work we will make branches off of here and then work -- for now

File: app/utils/get_filtered_records.py [hunk 1]
```diff
@@ -51,6 +53,24 @@ def _apply_agg_query(agg_field_exp: SQLalchemyExpression, label: str,
                 else_=cast(array([]), ARRAY(type))).label(label)
 
 
+def get_all_arbo_records():
```
Comment (harrietware) [inline app/utils/get_filtered_records.py:56]: Please add a filter to only get records where ETL Included = 1

File: app/arbo/arbo_etl/arbo_etl_main.py [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+    engine = create_engine('postgresql://{username}:{password}@{host_address}/whiteclaw'.format(
+        username=os.getenv('DATABASE_USERNAME'),
+        password=os.getenv('DATABASE_PASSWORD'),
+        host_address=os.getenv('DATABASE_HOST_ADDRESS')))
+
+    fields_mapping = {'Source Sheet': 'source_sheet',
+                      'Inclusion Criteria': 'inclusion_criteria',
+                      'Sample Start Date': 'sample_start_date',
+                      'Sample End Date': 'sample_end_date',
+                      'Age': 'age',
```
Comment (harrietware) [inline app/arbo/arbo_etl/arbo_etl_main.py:42]: Please add new field 'Age group'

File: app/serotracker_sqlalchemy/models.py [hunk 3]
```diff
@@ -238,3 +238,30 @@ class PopulationGroupOptions(db.Model):
...
+
+# ARBO
+
+class ArboRecords(db.Model):
+    __tablename__ = 'estimates'
+    __table_args__ = {'schema': 'Arbo'}
+
+    estimate_id = Column(BigInteger, primary_key=True, autoincrement=True)
+    sex = Column(String(1))
+    age = Column(String(64))
```
Comment (harrietware) [inline app/serotracker_sqlalchemy/models.py:251]: Add age_group

File: app/arbo/arbo_etl/arbo_etl_main.py [hunk 4]
```diff
@@ -0,0 +1,98 @@
...
+    load_dotenv()
+
+    estimate_table = Table(os.getenv('AIRTABLE_API_KEY'), os.getenv('AIRTABLE_ARBO_BASE_ID'), 'Study/Estimate Sheet')
+
+    engine = create_engine('postgresql://{username}:{password}@{host_address}/whiteclaw'.format(
+        username=os.getenv('DATABASE_USERNAME'),
+        password=os.getenv('DATABASE_PASSWORD'),
+        host_address=os.getenv('DATABASE_HOST_ADDRESS')))
+
+    fields_mapping = {'Source Sheet': 'source_sheet',
```
Comment (harrietware) [inline app/arbo/arbo_etl/arbo_etl_main.py:38]: Add 'State' for get_record_coordinates

**Pull request merged**